src/App.js

```diff
@@ -60,6 +80,11 @@ export default class App extends Component {
     this.setState({ selectedLanguage });
   }
 
+  onExampleChanged(e) {
+    const selectedExample = e.target.value;
```

Comment:
why add another variable declaration when `selectedExample` is almost as long as `e.target.value`?